docs/getting-started/function-triggers.md

```diff
@@ -0,0 +1,147 @@
+          "time": "2022-09-23T12:45:18.852Z",
+          "type": "boson.fn",
+          "source": "/boson/fn",
+          "specversion": "1.0",
+          "datacontenttype": "application/json",
+          "data": {
+            "message": "Hello World"
+          }
+        }
+    ```
+
+## Create a Broker
+
+Now, let's create the Broker. A Broker is a resource that will receive events
+from the event source and forward them to other resources.
+For many of the following commands, you will use the `kn` CLI. If you don't
+already have the `kn` CLI installed, please refer to the [Knative
+Client installation instructions](https://knative.dev/docs/client/install-kn/).
+
+Create a default Broker by running the command:
+
+```{ .console }
+kn broker create default
+```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+      Broker 'default' successfully created in namespace 'default'.
+    ```
+
+## Create a PingSource
+
+A PingSource is a Knative Event Source that generates events at a specified interval.
+To create the PingSource, you will also use the `kn` CLI. This command will
+result in a PingSource emitting an event every minute, sending it to the
+default Broker.
+
+Create a PingSource called, `ping` by running the command:
+
+```{ .console }
+kn source ping create ping --sink broker:default --schedule="*/1 * * * *"
+```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+    Ping source 'ping' created in namespace 'default'.
+    ```
+
+## Create a Trigger
+
+Now that the Broker and PingSource are created, you can create a Trigger that
+will forward all PingSource events to the function. In the following command,
+the `--sink` flag specifies your function as the destination for the CloudEvents.
+
+Create a Trigger by running the command:
+
+```{ .console }
+kn trigger create ping-trigger --sink hello
+```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+    Trigger 'ping-trigger' successfully created in namespace 'default'.
+    ```
+
+To see your function being invoked, you can use the `kubectl` CLI to watch the
+logs of the function pod. Watch the logs of the function pod by running the command:
+
+```{ .console }
+kubectl logs -l function.knative.dev=true --tail=10
+```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+    Received event
+    Context Attributes,
+      specversion: 1.0
+      type: dev.knative.sources.ping
+      source: /apis/v1/namespaces/default/pingsources/ping
+      id: d467df81-3296-48ed-bb1c-0662153f53b1
+      time: 2022-09-23T14:25:00.344067423Z
+    Extensions,
+      knativearrivaltime: 2022-09-23T14:25:01.295357548Z
+    ```
+
+That's it! To learn
+more about local development with Knative Functions, see
+[Next Steps with Knative Functions](../function-next-steps).
+Or view the [Command Line Reference](https://github.com/knative-sandbox/kn-plugin-func/blob/main/docs/reference/func.md)
+for detailed descriptions of all CLI commands.
```

Comment:
I think these should go in `/getting-started/next-steps.md`, not here (since this is in the middle of the tutorial flow, we don't want to explicitly route people out of the tutorial).